### ISSUE TITLE
can not map canjs main file

### ISSUE THREAD

**Issue by whitecolor** [stealjs/steal#379]
Want to remap (using `system.map` in `package.json`) main `can.js` file to my version that is located `baseURL/can.js` (for the purpose of redefining default rendering engine to `stache`)

so instead of loading `/node_modules/can/can.js` steal should load `/can.js`.

with old steal I just did:
```
System.config({
    paths: {
        'can/can': 'can.js'
    }
})
```

this does not work with new steal.

What config should I have? I tried different variants, but there wasn an error: something like:
```
TypeError: Error loading "package.json!npm" at http://localhost:3000/package.json
Cannot read property 'system' of undefined {stack: (...), message: "Error loading "package.json!npm" at http://localhot-…e.json↵Cannot read property 'system' of undefined"}message: "Error loading "package.json!npm" at http://localhost:3000/package.json↵Cannot read property 'system' of undefined"
```

**Comment by matthewp** [comment]
You should be able to do:

```js
"system": {
  map: {
    "can": "./can"
  }
}
```

And this does what's supposed to do but because `can` is also the name of a package it is then getting mapped back to the real can. So this is a bug.

**Comment by matthewp** [comment]
Being tracked here: https://github.com/stealjs/system-npm/issues/46

**Comment by matthewp** [comment]
Fixed in system-npm, reopening to remind myself to check once we've done a new steal patch release.

**Comment by matthewp** [comment]
This is fixed in 0.8.2, just released. I created an example repo where can is being mocked by a local file named can.js: https://github.com/matthewp/can-mock

**Comment by whitecolor** [comment]
Thanks that works.
